<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntellectSettingsViewClass</class>
 <widget class="QWidget" name="IntellectSettingsViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1">
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="styleSheet">
      <string notr="true">QFrame[Style_Widget=&quot;Group&quot;]
{
	border-radius:5px;
    border:2px solid #EEF2FB;
}
QLabel
{
	color:#6E6E6E;
}
QLabel[Style_Label=&quot;g&quot;]
{
	color:#000000;
}</string>
     </property>
     <property name="Style_Widget" stdset="0">
      <string>Style_White_Widget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="Style_Widget" stdset="0">
         <string>Group</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelUserName">
           <property name="text">
            <string>康复人：XX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="labelCaregiv">
           <property name="text">
            <string>护理人：XX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelBedNumber">
           <property name="text">
            <string>床号：XX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelNursingArea">
           <property name="text">
            <string>护理区：XX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelAge">
           <property name="text">
            <string>年龄：XX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labelAdmissionTime">
           <property name="text">
            <string>入住时间：XX</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="PixmapWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>用户管理</string>
              </property>
              <property name="Style_Label" stdset="0">
               <string>g</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QWidget" name="widget_14" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="btnUserLogin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUserEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUserRegister">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>注册</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUserExit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="Style_Widget" stdset="0">
         <string>Group</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCaregiver">
           <property name="text">
            <string>护理人ID：XXX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>密码：XXXXXX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelEngineer">
           <property name="text">
            <string>工程师：XXX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelMaintainer">
           <property name="text">
            <string>维护人ID：XXX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="PixmapWidget" name="widget_4" native="true">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>用户登陆</string>
              </property>
              <property name="Style_Label" stdset="0">
               <string>g</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QWidget" name="widget_15" native="true">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="btnCaregiverEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="Style_Widget" stdset="0">
         <string>Group</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelScreenBrightness">
           <property name="text">
            <string>屏幕亮度：10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelScreenOffTime">
           <property name="text">
            <string>屏幕关闭：10分钟</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="PixmapWidget" name="widget_6" native="true">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>屏幕设定</string>
              </property>
              <property name="Style_Label" stdset="0">
               <string>g</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QWidget" name="widget_16" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="btnScreenEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="Style_Widget" stdset="0">
         <string>Group</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelWifiName">
           <property name="text">
            <string>WIFI：123456789</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelWifiPassword">
           <property name="text">
            <string>WIFI密码：123456789</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="PixmapWidget" name="widget_10" native="true">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_28">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>网络设定</string>
              </property>
              <property name="Style_Label" stdset="0">
               <string>g</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QWidget" name="widget_17" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="btnWifiEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="Style_Widget" stdset="0">
         <string>Group</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCanPort">
           <property name="text">
            <string>CAN端口：9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="labelBaudRate485B">
           <property name="text">
            <string>波特率：9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelBaudRate485A">
           <property name="text">
            <string>波特率：9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelPort485A">
           <property name="text">
            <string>485-A端口：9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelBaudRate">
           <property name="text">
            <string>波特率：9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labelPort485B">
           <property name="text">
            <string>485-B端口：9600</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="PixmapWidget" name="widget_12" native="true">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_35">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>通讯设定</string>
              </property>
              <property name="Style_Label" stdset="0">
               <string>g</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QWidget" name="widget_18" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btnNetEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="Style_Widget" stdset="0">
      <string>Style_White_Widget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSlider" name="verticalSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PixmapWidget</class>
   <extends>QWidget</extends>
   <header>Component/PixmapWidget/PixmapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/QtIntellectNurse.qrc"/>
 </resources>
 <connections/>
</ui>
